### PR TITLE
Added section on how CI runs integration tests

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
The force-push to 13b7f4b8 added some clues in the section on CI that I got from @mauriciopoppe

**Review by cjwagner — APPROVED** [review]

File: contributors/devel/sig-testing/integration-tests.md [hunk 1]
```diff
@@ -76,3 +77,13 @@ make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS="
 
 If you set `KUBE_TEST_ARGS`, the test case will be run with only the `v1` API
 version and the watch cache test is skipped.
+
+### Integration testing in CI
+
+The CI job named `pull-kubernetes-integration`, which is defined in
+[https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/integration.yaml],
```
Comment (MadhavJivrajani) [inline contributors/devel/sig-testing/integration-tests.md:84]: Is this link supposed to be embedded in some text?
Perhaps something like [`integration.yaml`](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-testing/integration.yaml) (or something similar)?

File: contributors/devel/sig-testing/integration-tests.md [hunk 2]
```diff
@@ -76,3 +77,13 @@ make test-integration WHAT=./test/integration/pods GOFLAGS="-v" KUBE_TEST_ARGS="
...
+runs [k/k
+hack/jenkins/test-dockerized.sh](https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-dockerized.sh)
```
Comment (MadhavJivrajani) [inline contributors/devel/sig-testing/integration-tests.md:86]:
```suggestion
runs [`k/k/hack/jenkins/test-dockerized.sh`](https://github.com/kubernetes/kubernetes/blob/master/hack/jenkins/test-dockerized.sh)
```
small nit

**Review by MadhavJivrajani — COMMENTED** [review]
/hold
(please feel free to remove the hold after the `lgtm` label is applied!)

**Pull request closed without merging**